README.md

```diff
@@ -1 +1,289 @@
-# Hello-Microverse-Project-Two
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
```

Comment:
- [ ] Very good work setting up your first project 👏🏾. Please delete all comments in your code base, they make the code base dirty. `Line 25` clearly tells you what should be done.

**NOTE: Please remember to always delete all comments from your codebase**